File: 20221026/모범답안/20221026_02/Grading.js

/**
 * 
 * @param arr {number[]}
 * @returns number
 */
function solution(arr) {

    arr.sort((a, b) => a - b).reverse();
    const n = arr.length;

    const border = (n >> 1) - 1;
    let i = (n >> 1);

    while (i < n && arr[border] == arr[i]) {
        i += 1;
    }    

    return i;
}

File: 20221026/모범답안/20221026_02/Grading.py
def solution(arr):
    '''
    :param arr: int[]
    :return: int
    '''

    arr.sort(reverse=True)
    n = len(arr)


    border = n // 2 - 1
    i = n // 2
    while i < n and arr[border] == arr[i]:
        i += 1
    
    return i
